massgen/tests/test_srt_manager.py

```
@@ -0,0 +1,295 @@
+    path = mgr.write_settings_file(profile="execution", agent_id="agent_a")
+    assert path.exists()
+    data = json.loads(path.read_text())
+    assert "filesystem" in data and "network" in data
+    assert str(pm_with_paths["workspace"]) in data["filesystem"]["allowWrite"]
+
+
+# --------------------------------------------------------------------------- #
+# wrapping — single source of truth shared with the MCP server
+# --------------------------------------------------------------------------- #
+def test_wrap_command_string_form():
+    assert wrap_command_with_srt("echo hi", "/tmp/cfg.json") == "srt --settings /tmp/cfg.json sh -c 'echo hi'"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/cfg.json"

(S108)